velox/exec/Window.cpp

```diff
@@ -17,6 +17,8 @@
 #include "velox/exec/OperatorUtils.h"
 #include "velox/exec/Task.h"
 
+DEFINE_bool(SkipRowSortInWindowOp, false, "Skip row sort");
```

Comment:
This doesn't look right. To skip sorting, pass empty list of sorting keys to WindowNode. CC: @aditi-pandit